pkg/metrics/gatherer/utility.go

```diff
@@ -0,0 +1,74 @@
+package gatherer
+
+import (
+	"runtime"
+
+	"github.com/openshift/installer/pkg/metrics/timer"
+	"github.com/openshift/installer/pkg/version"
+)
+
+// InitializeInvocationMetrics is a utility function that adds the common information to create
+// an invocation metrics.
+func InitializeInvocationMetrics(metricName string) {
+	AddLabelValue(metricName, "result", "Success")
+	AddLabelValue(metricName, "returnCode", "1")
+	AddLabelValue(metricName, "os", runtime.GOOS)
+	CurrentInvocationContext = metricName
+}
+
+// LogError sets the result and returnCode to the right values and pushes the information to the
+// gateway. Must be called before the installer breaks execution.
+func LogError(err string, metricName string) {
+	AddLabelValue(metricName, "result", err)
+	AddLabelValue(metricName, "returnCode", "0")
```

Comment:
returnCode has been removed from the enhancement but typically 0 indicates success